### PR TITLE
Migrating MBI KB Content to DevDocs

### PR DESCRIPTION
**Pull request by erikmarr**
Reorganizing MBI API to include export API content that previously resided in the merchdocs site.

### CODE REVIEW

File: docs/export-api.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+group: mbi
+title: Automating Data Retrieval with the Data Export API
+zendesk_id: 360016732271
+functional_areas:
+  - Export API
+  - System
+  - Setup
+---
+
+The Magento BI API is provided to customers who wish to access some of the raw data behind their Magento BI dashboard.
```
Comment (keharper) [inline docs/export-api.md]: Make this paragraph more active voice. Perhaps the first sentence could be

The Magento BI API allows customers to access...

File: docs/export-api.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+The API is entirely HTTP-based. Methods that retrieve data must be accessed with a GET request, and those that create, modify or delete data with a POST request.
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
The API is entirely HTTP-based. Use a GET request to retrieve data or a POST request to create, modify, or delete data.
```

File: docs/export-api.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+### Authentication
+
+API connections are authenticated via a token that must be passed in the HTTP header of every request.
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
API connections are authenticated using a token that must be passed in the HTTP header of every request.
```

File: docs/export-api.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+1. Keys can be generated via **Export Data -> Export API**, and must be passed in the "X-RJM-API-Key" portion of every request header.
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
1. Generate a key from  **Export Data -> Export API**. You must specify this key in the "X-RJM-API-Key" header of every request.
```

Comment (keharper) [reply docs/export-api.md]: Also, these steps should be bullet items.

File: docs/export-api.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+1. All keys must be associated with a single client and a set of IP addresses that the requests will be made from.  IP addresses can be specified either as a specific address, or as a range of addresses in [CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).  For example, the CIDR notation to allow ALL IP addresses would be: "0.0.0.0/0".
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
1. All keys must be associated with a single client and a set of IP addresses that are allowed to make requests.  IP addresses can be specified either as a specific address, or as a range of addresses in [CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).  For example, the CIDR notation to allow ALL IP addresses would be: "0.0.0.0/0".
```

**Review by keharper — COMMENTED** [review]
A few comments. I'll complete another review later.

File: docs/export-api.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+Methods are defined by an HTTP Verb (GET, POST, PUT, DELETE), a URL and a set of arguments. The response HTTP status code will be 200-Success for all successful requests unless otherwise specified.
+
+### Exporting Raw Data Exports
+
+{:.bs-callout-info}
+You need a [raw data export](https://docs.magento.com/mbi/tutorials/export-raw-data.html) before calling the following methods.
+
+**Endpoint**
+
+`GET /export \[no arguments\]`
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
`GET /export`
```

File: docs/export-api.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+
+API connections are authenticated using a token that must be passed in the HTTP header of every request.
+
+* Generate a key from **Export Data -> Export API**. You must specify this key in the `X-RJM-API-Key` header of every request.
+
+* All keys must be associated with a single client and a set of IP addresses that are allowed to make requests. IP addresses can be specified either as a specific address, or as a range of addresses in [CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). For example, the CIDR notation to allow ALL IP addresses would be: `0.0.0.0/0`.
+
+![mbi-valid-ip-address.png](../docs/images/mbi-valid-ip-address.png)
+
+## Methods
```
Comment (keharper) [inline docs/export-api.md]:
```suggestion
## HTTP Methods
```

File: docs/export-api.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+
+Use a GET request to retrieve data or a POST request to create, modify, or delete data.
+
+Methods are defined by an HTTP Verb (GET, POST, PUT, DELETE), a URL and a set of arguments. The response HTTP status code will be 200-Success for all successful requests unless otherwise specified.
+
+### Exporting Raw Data Exports
```
Comment (keharper) [inline docs/export-api.md]: Do all the MBI headings use init caps for all significant words?

File: docs/export-api.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+group: mbi
+title: Magento BI Export API
+zendesk_id: 360016732271
+functional_areas:
+  - Export API
+  - System
+  - Setup
+---
+​
+The Magento BI Export API allows you to  to access some of the raw data behind your Magento BI dashboard.
```
Comment (je55ek) [inline docs/export-api.md]: double "to" in line 11

File: docs/export-api.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+Before using the import API, make sure you [authenticate](../docs/getting-started.md#authentication) your connection.
```
Comment (je55ek) [inline docs/export-api.md]: "import" should read "export"

**Review by je55ek — APPROVED** [review]

**Review by keharper — APPROVED** [review]

File: docs/export-api.md (hunk 8 above)
Comment (erikmarr) [reply docs/export-api.md]: I'm sure we're inconsistent throughout. Will address in a future clean up.

**Pull request merged**